dev/com.ibm.ws.springboot.fat20.concurrency.app/src/main/java/com/example/demo/ConcurrencyApplicationConfig.java

```diff
@@ -0,0 +1,70 @@
+package com.example.demo;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.naming.NamingException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.DefaultManagedTaskExecutor;
+import org.springframework.scheduling.concurrent.DefaultManagedTaskScheduler;
+import org.springframework.stereotype.Service;
+
+@Configuration(proxyBeanMethods = false)
```

Comment:
I am not sure what to make of this because I thought to have `@Async` methods the bean has to be proxied.  I suggest this be removed.